### DISCUSSION TITLE
API (failed)net::ERR_CONNECTION_REFUSED

### DISCUSSION THREAD

**Discussion by jarzebski** [blakeblackshear/frigate#6490] [General]
Hi, I'm trying build dev version Frigate. I run frigate and web app, but not loading. Frontend say 

http://localhost:5000/api/config -> (failed)net::ERR_CONNECTION_REFUSED on request localhost:5000

Vite runs on 5173 port

**Comment by NickM-27** [comment]
You'll need to give more info of what commands you have run and what vscode setup you are using.

**Reply by jarzebski** [reply]
I'm running frigate via command : python3 -m frigate (app run correctly)
next in web directory I run npm install and npm run dev.
all dependencies is installed correctly - but when i'm view in browser on 5173 I see frigate APP, but only spinner show up and in console log, showing error to connect /ap/config on port 5000

for  now I'm not use vscode - all commands fired from unix cosnole

**Reply by NickM-27** [reply]
you may need to run nginx manually in another terminal tab
